src/PhpFlo/Common/Test/TestUtilityTrait.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/*
+ * This file is part of the phpflo/phpflo-flowtrace package.
+ *
+ * (c) Marc Aschmann <maschmann@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+declare(strict_types=1);
+namespace PhpFlo\Common\Test;
+
+use PhpFlo\Common\ComponentInterface;
+use PhpFlo\Core\Interaction\InternalSocket;
+
+/**
+ * Mock class helper.
+ *
+ * @package PhpFlo\Common\Test
+ * @author Marc Aschmann <maschmann@gmail.com>
+ */
+trait TestUtilityTrait
+{
+    /**
```

Comment:
Not sure if needed: If someone wants to write tests for components he should look into the component repo and there for the tests